pkg/storage/export/export/export.go

```diff
@@ -456,6 +456,10 @@ func (ctrl *VMExportController) handleSource(vmExport *exportv1.VirtualMachineEx
 	if err != nil {
 		return 0, err
 	}
+	if vmExport == nil {
```

Comment:
I set it to nil to spare some updates and stuff in case we deleted the resource
https://github.com/kubevirt/kubevirt/blob/4744a58f368d882bf7df83be02207fdb7ca0ecfc/pkg/storage/export/export/export.go#L521